Franka Hand Binary Not found

## Type of Issue

Select the type of issue:
- [x] Bug report (to report a bug)
- [ ] Feature request (to request an additional feature)
- [ ] Tracker (I am just using this as a tracker)
- [ ] Refactor request
- [ ] Documentation Ask

## Description
First off, thank you all for a well built overall repo!

I've gotten polymetis to work with a Franka Panda Robot I am using, but I cannot use the Franka Gripper. when I follow the directions of the local install, and then simply run `launch_gripper.py gripper=franka_hand` I am told the franka_hand_client binary cannot be found, despite the fact that building libfranka and polymetis succeeded. 
## Current Behavior
running  `launch_gripper.py gripper=franka_hand` 
leads to following error
```
Warning: Failed to load 'libtorchscript_pinocchio.so' from CONDA_PREFIX, loading from default build directory instead: '/home/panda5/fairo/polymetis/polymetis/build/torch_isolation/libtorchscript_pinocchio.so'
Warning: Failed to load 'libtorchrot.so' from CONDA_PREFIX, loading from default build directory instead: '/home/panda5/fairo/polymetis/polymetis/build/torch_isolation/libtorchrot.so'
/home/panda5/anaconda3/envs/polymetis-local/lib/python3.8/site-packages/hydra/plugins/config_source.py:190: UserWarning: 
Missing @package directive gripper/franka_hand.yaml in pkg://polymetis.conf.
See https://hydra.cc/docs/next/upgrades/0.11_to_1.0/adding_a_package_directive
  warnings.warn(message=msg, category=UserWarning)
Gripper server running at 0.0.0.0:50052.
[2022-10-03 19:18:56,228][polymetis.robot_client.executable_robot_client][INFO] - === Config: ===
exec: franka_hand_client
robot_ip: 172.16.0.2
control_ip: 0.0.0.0
control_port: 50052
robot_client_metadata_path: /tmp/tmphrzoo0zg

Traceback (most recent call last):
  File "/home/panda5/fairo/polymetis/polymetis/python/scripts/launch_gripper.py", line 40, in main
    gripper_client.run()
  File "/home/panda5/fairo/polymetis/polymetis/python/polymetis/robot_client/executable_robot_client.py", line 69, in run
    assert path_to_exec, f"Unable to find binary {self.executable_cfg.exec}"
AssertionError: Unable to find binary franka_hand_client

```

## Expected Behavior

for the server to start without issues
## Steps to reproduce

I created a clean conda env, directly following all the steps for the local installation here:  https://facebookresearch.github.io/fairo/polymetis/installation.html. The only thing I changed was when running the install libfranka script, I ran `./scripts/build_libfranka.sh 0.8.0` because this is the version that happens to work with my NUC /RT_kernel spec

Do you mean that even after `franka_hand_client` exists in the build directory, you are still getting the same `AssertionError: Unable to find binary franka_hand_client` error trying to run `launch_gripper.py gripper=franka_hand`?

--- a/polymetis/polymetis/python/polymetis/utils/data_dir.py
+++ b/polymetis/polymetis/python/polymetis/utils/data_dir.py
@@ -9,6 +9,7 @@ import polymetis
 
 PKG_ROOT_DIR = polymetis.__path__[0]
 DATA_DIR = os.path.join(PKG_ROOT_DIR, "data")
+BUILD_DIR = os.path.abspath(os.path.join(PKG_ROOT_DIR, "..", "..", "build"))
 
 
 def get_full_path_to_urdf(path: str):

--- a/polymetis/polymetis/python/scripts/launch_gripper.py
+++ b/polymetis/polymetis/python/scripts/launch_gripper.py
@@ -6,17 +6,23 @@
 # LICENSE file in the root directory of this source tree.
 
 import os
-import signal
 import time
+import logging
 
 import hydra
 
 from polymetis.robot_servers import GripperServerLauncher
 from polymetis.utils.grpc_utils import check_server_exists
+from polymetis.utils.data_dir import BUILD_DIR
+
+log = logging.getLogger(__name__)
 
 
 @hydra.main(config_name="launch_gripper")
 def main(cfg):
+    log.info(f"Adding {BUILD_DIR} to $PATH")
+    os.environ["PATH"] = BUILD_DIR + os.pathsep + os.environ["PATH"]
+
     if cfg.gripper:
         pid = os.fork()
     else:

--- a/polymetis/polymetis/python/scripts/launch_robot.py
+++ b/polymetis/polymetis/python/scripts/launch_robot.py
@@ -6,7 +6,6 @@
 # LICENSE file in the root directory of this source tree.
 
 import os
-import socket
 import logging
 import subprocess
 import atexit
@@ -17,7 +16,7 @@ import signal
 import hydra
 
 from polymetis.utils.grpc_utils import check_server_exists
-from polymetis.utils.data_dir import PKG_ROOT_DIR, which
+from polymetis.utils.data_dir import BUILD_DIR, which
 
 
 log = logging.getLogger(__name__)
@@ -25,9 +24,8 @@ log = logging.getLogger(__name__)
 
 @hydra.main(config_name="launch_robot")
 def main(cfg):
-    build_dir = os.path.abspath(os.path.join(PKG_ROOT_DIR, "..", "..", "build"))
-    log.info(f"Adding {build_dir} to $PATH")
-    os.environ["PATH"] = build_dir + os.pathsep + os.environ["PATH"]
+    log.info(f"Adding {BUILD_DIR} to $PATH")
+    os.environ["PATH"] = BUILD_DIR + os.pathsep + os.environ["PATH"]
 
     # Check if another server is alive on address
     assert not check_server_exists(

